src/index.html

```diff
@@ -1,14 +1,43 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta charset="UTF-8" >
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com" >
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
+    <link rel="stylesheet" href="./style.css" >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <a href="#">
+          <img src="./images/logo.png" alt="logo" >
+        </a>
+        <nav class="nav">
+          <ul class="navorder">
+            <li class="navelement"><a href="#" class="is-active">Apple</a></li>
+            <li class="navelement"><a href="#" class="is-active">Samsung</a></li>
+            <li class="navelement"><a href="#" class="is-active">Smartphones</a></li>
+            <li class="navelement">
+              <a href="#" class="is-active"
+              data-qa="hover">Laptops & Computers</a>
+            </li>
+            <li class="navelement"><a href="#" class="is-active">Gadgets</a> </li>
```

Comment:
[Add empty lines between multiline sibling blocks of HTML](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#html-formatting)

![image](https://github.com/mate-academy/layout_moyo-header/assets/43861663/9b194817-9567-4a48-9c78-e664deb5d25d)